integration-tests/build_run_cluster.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+echo $DRUID_INTEGRATION_TEST_OVERRIDE_CONFIG_PATH
+
+export DIR=$(cd $(dirname $0) && pwd)
+export HADOOP_DOCKER_DIR=$DIR/../examples/quickstart/tutorial/hadoop/docker
+export DOCKERDIR=$DIR/docker
+export SERVICE_SUPERVISORDS_DIR=$DOCKERDIR/service-supervisords
+export ENVIRONMENT_CONFIGS_DIR=$DOCKERDIR/environment-configs
+export SHARED_DIR=${HOME}/shared
+export SUPERVISORDIR=/usr/lib/druid/conf
+export RESOURCEDIR=$DIR/src/test/resources
+
+# so docker IP addr will be known during docker build
+echo ${DOCKER_IP:=127.0.0.1} > $DOCKERDIR/docker_ip
+
+if !($DRUID_INTEGRATION_TEST_SKIP_BUILD_DOCKER); then
+  sh ./script/copy_resources.sh
+  sh ./script/docker_build_containers.sh
+fi
+
+if !($DRUID_INTEGRATION_TEST_SKIP_RUN_DOCKER); then
+  sh ./stop_cluster.sh
```

Comment:
stop_cluster.sh has a check for DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER
The check for DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER is too skip teardown of the cluster after the test finish running (for debugging). In this case, where we are stopping the cluster so that we can start then fresh to run integration test, we should skip checking DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER here.

Reply:
This block is the same as we have in master, I only removed dublicated code. see: run_cluster.sh#line:24
We need to stop container 2 times:
1. before - if you have already runned druid cluster and you want to run it again then you need to stop it
2. after - stop druid containers after integration tests

Also step (1) needed in case if your tests failed then step (2) not executed

Reply:
That is correct. 
I think we want to stop container in number 1 if DRUID_INTEGRATION_TEST_SKIP_START_DOCKER is not true and we want to stop container in number 2 if DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER is not false.

However, in number 1, we do not have to care about what is the value of DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER. For example, number 1 we should stop (so that we can run before starting test) if DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER=true and DRUID_INTEGRATION_TEST_SKIP_START_DOCKER=false

Reply:
In other words, the DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER only applies to number 2 not number 1

Reply:
The purpose of DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER is only to prevent the tearing down of Druid cluster after the test completed. This is so that you can continue to inspect the cluster after the test finished.

Reply:
I am cunfules a little bit about do we need to change something or not.
With new updates in pull request:
We dont have "DRUID_INTEGRATION_TEST_SKIP_STOP_DOCKER" variable.
We skip stop step if we DRUID_INTEGRATION_TEST_SKIP_START_DOCKER=true

Reply:
I think it is fine as is in the updated patch